packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -548,8 +548,7 @@ MeetingUtil.setPeerConnectionEvents = (meeting) => {
   // In case ICE fail
   const {peerConnection} = meeting.mediaProperties;
 
-  peerConnection.oniceconnectionstatechange = (ev) => {
-    console.log('ICE CHANGE ', ev);
```

Comment:
oops must've missed these last 2 when I did the cleanup. You can also send them to LoggerProxy.trace, which is kind of what I would prefer, but both of these logs are fairly noisy.